modules/mw.cx.init.Translation.js

```
@@ -260,6 +272,14 @@ mw.cx.init.Translation.prototype.attachToDOM = function (veTarget) {
  * @return {Promise}
  */
 mw.cx.init.Translation.prototype.fetchSourcePageContent = function (wikiPage, targetLanguage, siteMapper) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)